backend/tracim_backend/config.py

```diff
@@ -681,9 +682,22 @@ def __load_uploaded_files_config(self) -> None:
         )
 
     def _load_live_messages_config(self) -> None:
-        self.LIVE_MESSAGES__CONTROL_ZMQ_URI = self.get_raw_config(
-            "live_messages.control_zmq_uri", "tcp://localhost:5563"
+        self.LIVE_MESSAGES__CONTROL_ZMQ_URI = string_to_list(
+            self.get_raw_config("live_messages.control_zmq_uri", "tcp://localhost:5563"),
+            cast_func=str,
+            separator=",",
+        )
+        self.LIVE_MESSAGES__PUSH_ZMQ_URI = string_to_list(
+            self.get_raw_config("live_messages.push_zmq_uri"),
+            cast_func=str,
+            separator=",",
         )
+        self.LIVE_MESSAGES__PUB_ZMQ_URI = string_to_list(
+            self.get_raw_config("live_messages.pub_zmq_uri"),
+            cast_func=str,
+            separator=",",
+        )
+
```

Comment:
Here we should assert:

- first case : only CONTROL_URI is setup
- second case : all parameters are fully filled

if len( self.LIVE_MESSAGES__PUSH_ZMQ_URI) > 0:  # all parameters are fully filled
    assert len(self.LIVE_MESSAGES__PUSH_ZMQ_URI) == len(self.LIVE_MESSAGES__PUB_ZMQ_URI) and len(self.LIVE_MESSAGES__PUB_ZMQ_URI) == len(self.LIVE_MESSAGES__CONTROL_ZMQ_URI) and len(self.LIVE_MESSAGES__CONTROL_ZMQ_URI) > 1,  "CONTROL, PUSH and PUB ZMQ URIs must be configured at the same time"

if len( self.LIVE_MESSAGES__PUSH_ZMQ_URI) == 0:  #only control is setup
    assert len(self.LIVE_MESSAGES__PUSH_ZMQ_URI) == len(self.LIVE_MESSAGES__PUB_ZMQ_URI) and len(self.LIVE_MESSAGES__PUSH_ZMQ_URI) == 0,  "PUSH and PUB ZMQ URIs must be "not configured" at the same time"

```

Reply:
@Millefeuille42

Reply:
done in last commit